### ISSUE TITLE
Ability to work in 'singleton sandbox' mode for each blueprint

### ISSUE THREAD

**Issue by doppleware** [QualiSystemsLab/colony-cli#32]
Instead of requiring you to end a previous sandbox, when you create a new sandbox, the CLI will automatically end the previously running blueprint (async send end signal) before starting the new one.

Allow overriding this behavior in the config file or via cli flag.

**Comment by ddovbii** [comment]
> Instead of requiring you to end a previous sandbox

Have I missed something? Did we add some feature requiring users to end recently launched sandbox before starting a new one?

**Comment by doppleware** [comment]
Its a feature request not a bug (I labeled it as "enhancement")

**Comment by ddovbii** [comment]
Yes, I understood it's an enhancement. I'm just not sure that I followed the idea

**Comment by doppleware** [comment]
The idea is that as a developer I won't need to track and end the old sandboxes, I'll have one 'debugging' session per BP
